pyproject.toml

```diff
@@ -48,7 +53,7 @@ dependencies = [
 rdt = { main = 'rdt.cli.__main__:main' }
 
 [project.optional-dependencies]
-copulas = ['copulas>=0.12.0',]
+copulas = ['copulas@git+https://git@github.com/sdv-dev/copulas.git',]
```

Comment:
@rwedge Can we point this to the [copulas release candidate](https://pypi.org/project/copulas/0.12.1.dev0/) to see if it passes?